File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        res = []
        for i in nums:
            if i in count:
                count[i] += 1
            else:
                count[i] = 1
        j = sorted(count.values(), reverse=True)
        kth = j[k - 1]
        for num, freq in count.items():
            if freq >= kth and len(res) < k:
                res.append(num)
        return res

File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
    for (let i=0;i<nums.length;i++){
    let k=nums[i]
    let j=i+1
    for(j;j<nums.length;j++){
     let m=nums[j]
     if(k==m){
    return true
}}}
    
    return false}
}
